btree/btree.c

```diff
@@ -3112,6 +3112,46 @@ static void ff_rec_del_credit(const struct nd *node, m0_bcount_t ksize,
  * to Value close to the corresponding Key as it is done in the leaf node.
  *
  */
+/**
+ * CRC Support:
+ *
+ * In btree context, CRC will be used to verify the btree-node data read from
+ * storage. It can be used to verify keys and/or values stored in the node. An
+ * additional CRC(known as CRC of CRCs) will be computed from all the
+ * individual CRCs and stored in the node header.
+ *
+ * There are two proposal for supporting CRCs in the node:
+ * 1. Discrete CRC : Calculate CRC for each key and/or value and store it after
+ * the respective key in the node. In case CRC is provided by
+ * user as part of key and/or value, there is no need of calculating CRCs. The
```

Comment:
on't we want to calculate CRC irrespective of user-provided CRC. As if we do downgrade or upgrade, the user provided CRC length may get differ, so we may not be able to calculate cCRC accurately.